### DISCUSSION TITLE
Is sui subscribe method "suix_subscribeTransaction" working?

### DISCUSSION THREAD

**Discussion by Arthur0323** [MystenLabs/sui#19412] [Q&A]
hey guys, I`m integrating SUI blockchain using getBlock.io for rpc calls, everything is working fine, except web socket calls, I subscribe 'suix_subscribeTransaction' then filter by 'ToAddress' for checking deposites. but it doesn`t work.

**Comment by NgocChung1998** [comment]
same err

**Comment by goodylili** [comment]
same error
